### DISCUSSION TITLE
Commerce Equipment YAMAHA STAGEPAS 200

### DISCUSSION THREAD

**Discussion by Pighome7** [J-tour/J-tour.github.io#22] [Polls]
#23 issue ref.
### YAMAHA產品合作意願
_Equipment_ STAGEPAS 200

![5200a6a561d243f6d7fec24f5a9bf83e](https://github.com/J-tour/J-tour.github.io/assets/163790388/c809e7e0-6e5a-4197-a3b8-fc96046ac75a)

> YAMAHA STAGEPAS 200 Amplifier

### Info

1. 最大180W的輸出音量
1. 內建PA系統(5軌、有效果器)
1. 內建鋰電池可不插電使用(約10小時)
1. 有藍芽及APP可以連接調整參數

### Link
[產品資訊](https://tw.yamaha.com/zh/products/proaudio/pa_systems/stagepas_200/index.html)
[產品街頭實測（在4:12分以後）](https://www.youtube.com/watch?v=z69R2GKHHBw)

**Comment by ShengHaoLo** [comment]
我覺得這個產品不錯，也的確可以省下不小的開支，如果只需要授權個影片什麼的倒是沒問題，還可以增加能見度

**Comment by Pighome7** [comment]
我認為YAMAHA的產品可以信任，這款音箱我看了自己其實也蠻心動的，功能也很不錯

**Comment by xinyannn1** [comment]
同意合作🙌

**Comment by Pighome7** [comment]
票數已過半，將進入下一階段的討論。

**Comment by ShengHaoLo** [comment]
#23 Shehbey YAMAHA STAGEPAS 200 issue conversery
